test-cases/artifacts/ubuntu2204.yaml

```diff
@@ -0,0 +1,23 @@
+backtrace_decoding: false
+cluster_backend: 'gce'
+gce_image_db: 'https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-2204-lts'
+gce_instance_type_db: 'n1-standard-2'
+gce_root_disk_type_db: 'pd-ssd'
+gce_root_disk_size_db: 50
+gce_n_local_ssd_disk_db: 1
+instance_provision: "spot"
+instance_provision_fallback_on_demand: true
+logs_transport: 'ssh'
+n_db_nodes: 1
+n_loaders: 0
+n_monitor_nodes: 0
+nemesis_class_name: 'NoOpMonkey'
+scylla_linux_distro: 'ubuntu-jammy'
+scylla_repo: 'http://downloads.scylladb.com/unstable/scylla/master/deb/unified/latest/scylladb-master/scylla.list'
+scylla_apt_keys:
+  # When will use manager 2.6, need to remove this commit because it's a workaround to make artifacts works
+  - '5E08FBD8B5D6EC9C'  # ScyllaDB Package Signing Key 2020 <security@scylladb.com>
```

Comment:
I think we don't need this one, since it's for the older versions.
But please test.

Reply:
@KnifeyMoloko did you remove both keys?
We must need at least one of them.

Reply:
Sorry, a bit to eager with the removals. I left the 2022 Signing Key now.